### ISSUE TITLE
[TheScene] ERROR: Unable to extract info json

### ISSUE THREAD

**Issue by Hrxn** [rg3/youtube-dl#12235]
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2017.02.22**
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

TheScene IE probably broken again
https://thescene.com/watch/vogue/march-2017-cover-video-vogue-kendall-jenner-gigi-hadid

---
```
F:\Incoming\Media>youtube-dl --ignore-config -v "https://thescene.com/watch/vogue/march-2017-cover-video-vogue-kendall-jenner-gigi-hadid"
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--ignore-config', '-v', 'https://thescene.com/watch/vogue/march-2017-cover-video-vogue-kendall-jenner-gigi-hadid']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2017.02.22
[debug] Python version 3.4.4 - Windows-10-10.0.14393
[debug] exe versions: ffmpeg N-83152-gf7e9275, ffprobe N-83152-gf7e9275
[debug] Proxy map: {}
[TheScene] march-2017-cover-video-vogue-kendall-jenner-gigi-hadid: Downloading webpage
[TheScene] march-2017-cover-video-vogue-kendall-jenner-gigi-hadid: Downloading webpage
ERROR: Unable to extract info json; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkf42q19b\build\youtube_dl\YoutubeDL.py", line 704, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkf42q19b\build\youtube_dl\extractor\common.py", line 427, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkf42q19b\build\youtube_dl\extractor\thescene.py", line 35, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkf42q19b\build\youtube_dl\extractor\common.py", line 759, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract info json; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.


F:\Incoming\Media>
```

---

Extractor used to work before, I assume site got changed in the meantime.

**Comment by ThomasChr** [comment]
Fixed in PR #12245

**Comment by Hrxn** [comment]
Thanks a lot, guys! 🍾 🥂
